overlord/hookstate/hookstate_test.go

```diff
@@ -289,6 +289,39 @@ func (s *hookManagerSuite) TestHookTaskEnforcesTimeout(c *C) {
 	checkTaskLogContains(c, s.task, `.*exceeded maximum runtime of 200ms`)
 }
 
+func (s *hookManagerSuite) TestHookTaskEnforcesDefaultTimeout(c *C) {
+	restore := hookstate.MockDefaultHookTimeout(150 * time.Millisecond)
+	defer restore()
+
+	// Force the snap command to hang
+	s.command = testutil.MockCommand(c, "snap", "while true; do sleep 1; done")
+
+	s.manager.Ensure()
+	completed := make(chan struct{})
+	go func() {
```

Comment:
I see the other tests are like this, but naively I'm not quite sure why need the goroutine here